test/test.js

```diff
@@ -81,50 +82,54 @@ function(assert) {
     var state = document.getElementById(id)
       .getElementsByClassName('count')[0].innerHTML;
     console.log('state:', state);
-    assert.equal(state, 8); // model was incremented successfully
+    assert.equal(state, 8, 
+    "End State is 8 after incrementing 7 by 1 (as expected)"); // model was incremented successfully
     done();
   });
 
   btn.dispatchEvent(evt);
   // document.getElementById(id).getElementsByClassName('inc')[0].click();
 });
 
-// test('Click reset button resets state to 0', function(assert) {
-//   mount(7, update, view, id);
-//   var root = document.getElementById(id);
-//   assert.equal(root.getElementsByClassName('count')[0].textContent, 7);
-//   var btn = root.getElementsByClassName("reset")[0]; // click reset button
-//   btn.click(); // Click the Reset button!
-//   var state = root.getElementsByClassName('count')[0].textContent;
-//   assert.equal(state, 0); // state was successfully reset to 0!
-//   empty(root); // clean up after tests
-//   // console.log('STATE:', state);
-// });
+test('Click reset button resets state to 0', function(assert) {
+  mount(7, update, view, id);
+  var root = document.getElementById(id);
+  assert.equal(root.getElementsByClassName('count')[0].textContent, 7);
+  var btn = root.getElementsByClassName("reset")[0]; // click reset button
+  btn.click(); // Click the Reset button!
+  var state = root.getElementsByClassName('count')[0].textContent;
+  assert.equal(state, 0, "State is 0 (Zero) after reset."); // state was successfully reset to 0!
+  empty(root); // clean up after tests
+  // console.log('STATE:', state);
```

Comment:
There are a couple of commented out `console.logs` but I'm happy to leave them in for the time being as these might actually be useful to beginners when we publicise the tutorial #29